lib/beeswaxapi/endpoints/preset.rb

```diff
@@ -0,0 +1,5 @@
+module BeeswaxAPI
+  class Preset < Endpoint
+    path :"v2/presets"
```

Comment:
Just to double check - since this is going to be used as a singular action (GET `v2/presets/<id>`) I made the class name singular but the path plural (presets). Is that correct?

Reply:
Why not use it for the list action too?

Reply:
oh yeah I intended that it would be possible to use it for both. As far as I can see the convention in the gem is to keep the class name singular though (but set the path name as plural). Does it look correct to you?